deploy/docker/watchtower-hooks/pre-update.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+appsmithctl backup --error_mail || exit 1
```

Comment:
This is `--error-mail`, right?

```suggestion
appsmithctl backup --error-mail || exit 1
```